h2/src/main/org/h2/table/Column.java

```diff
@@ -523,14 +510,7 @@ private String getCreateSQL(boolean includeName) {
                 buff.append('(').append(precision).append(", ").append(scale).append(')');
                 break;
             case Value.ENUM:
-                buff.append('(');
-                for (int i = 0; i < enumerators.length; i++) {
-                    buff.append('\'').append(enumerators[i]).append('\'');
-                    if(i < enumerators.length - 1) {
-                        buff.append(',');
-                    }
-                }
-                buff.append(')');
+                buff.append(extTypeInfo.toString());
```

Comment:
using toString() for anything other than debugging is a little dodgy, rather add a getCreateSql() method for  ExtTypeInfoEnum

Reply:
OK.

I also found a regression `TestSpatial`, I missed it somehow when I tested this earlier.